plugins/woocommerce/tests/e2e-pw/README.md

```diff
@@ -19,6 +19,8 @@ This is the documentation for the new e2e testing setup based on Playwright and
 
 ## Pre-requisites
 
+In case this is the first time building WooCommerce, it is advised to go through the [WooCommerce Monorepo prerequisites](https://github.com/woocommerce/woocommerce/blob/trunk/README.md#prerequisites) first.
```

Comment:
I think what @adimoldovan meant in [his comment](https://github.com/woocommerce/woocommerce/pull/43552#discussion_r1452270753) about relative path is to avoid using absolute path with `https://github.com/...` and instead point to the .md file with something like `../../../../README.md#prerequisites`. More info about relative paths [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#:~:text=Relative%20links%20are%20easier%20for%20users%20who%20clone%20your%20repository.%20Absolute%20links%20may%20not%20work%20in%20clones%20of%20your%20repository%20%2D%20we%20recommend%20using%20relative%20links%20to%20refer%20to%20other%20files%20within%20your%20repository.).  Using relative paths also means that documentation can stand on its own without referencing, in our case, to GitHub. 

Since we're using an absolute path in the rest of this README file, I would say it's okay to leave it now with `https://github.com/...` and change all paths as a part of separate PR in the future.

Reply:
I like the idea of relative paths based on the link Ivan shared. @Stojdza - are you ok to create an Issue in our [quality repo](https://github.com/woocommerce/woocommerce-quality/issues) to update this? 🙂

Reply:
gh-woocommerce/woocommerce-quality

Reply:
Done - https://github.com/woocommerce/woocommerce-quality/issues/761